public/app/plugins/panel/news/NewsPanelEditor.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { PureComponent } from 'react';
+import { FormField, PanelOptionsGroup } from '@grafana/ui';
+import { PanelEditorProps } from '@grafana/data';
+import { NewsOptions } from './types';
+import { Switch } from '@grafana/ui';
+
+interface State {
+  feedUrl: string;
+}
+
+export class NewsPanelEditor extends PureComponent<PanelEditorProps<NewsOptions>, State> {
+  constructor(props: PanelEditorProps<NewsOptions>) {
+    super(props);
+
+    this.state = {
+      feedUrl: props.options.feedUrl,
+    };
+  }
+
+  onUpdatePanel = () =>
+    this.props.onOptionsChange({
+      ...this.props.options,
+      feedUrl: this.state.feedUrl,
+    });
+
+  onFeedUrlChange = ({ target }: any) => this.setState({ feedUrl: target.value });
+
+  onProxyChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    this.props.onOptionsChange({
+      ...this.props.options,
+      proxy: e.target.checked,
+    });
+  };
+
+  render() {
+    const { feedUrl } = this.state;
+    const { proxy } = this.props.options;
+
+    return (
+      <>
+        <PanelOptionsGroup title="Feed">
+          <div className="gf-form">
+            <FormField
```

Comment:
One super small improvement could be to use the new `Forms` components located in `@grafana-ui/Forms/*`

Reply:
@mckn not yet unfortunately. This is Panel edit and I think we want to keep it consistent with other panel edits.

Reply:
OK ok, sweet! :)